### PR TITLE
updated documentation readme file

### PR DESCRIPTION
**Pull request by vijayshiyani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,279 +46,412 @@ Sample demo setp by step javascript example are avalible at [Demo Folder][demo-l
 -   OPERATOR_ID=0.0.xxxx
```
Comment (emurova) [inline README.md:46]: Hi @vijayshiyani ,
its typo in line 44 `setp`
Thanks

**Comment by emurova** [comment]
Hey @vijayshiyani and @linasi ,
can we use a linter? At least it would be good to fix the very long lines :-)
What do you think?
Thanks

File: README.md (hunk 1 above)
Comment (vijayshiyani) [reply README.md]: fixed

**Comment by vijayshiyani** [comment]
> Hey @vijayshiyani and @linasi , can we use a linter? At least it would be good to fix the very long lines :-) What do you think? Thanks

I have install https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint and it works.

**Comment by emurova** [comment]
> > Hey @vijayshiyani and @linasi , can we use a linter? At least it would be good to fix the very long lines :-) What do you think? Thanks
> 
> I have installed https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint and it works.

Im talking about ESLint or TSLint.

**Review by linasi — APPROVED** [review]
👍

**Pull request merged**